packages/edit-site/src/components/global-styles/use-global-styles-output.js

```diff
@@ -283,7 +284,8 @@ export function getStylesDeclarations(
 				return;
 			}
 		}
-		output.push( `${ cssProperty }: ${ compileStyleValue( ruleValue ) }` );
+
+		output.push( `${ cssProperty }: ${ rule.value }` );
```

Comment:
Does using `ruleValue` vs `rule.value` make a difference here?

Reply:
Yes! Good spotting. That was a bad rebase. Fixing...